test/k8sT/Services.go

```diff
@@ -93,8 +95,12 @@ var _ = Describe("K8sServicesTest", func() {
 			}
 		}
 
-		k8s1NodeName, k8s1IP = kubectl.GetNodeInfo(helpers.K8s1)
-		k8s2NodeName, k8s2IP = kubectl.GetNodeInfo(helpers.K8s2)
+		k8s1NodeName, k8s1InternalIP = kubectl.GetNodeInfo(helpers.K8s1)
```

Comment:
Nit: can we make `GetNodeInfo` to return both IP addrs?

Reply:
Fixed.